### ISSUE TITLE
Need to update api integration, V2 now deprecated on Twitch VODs

### ISSUE THREAD

**Issue by MatthewCroughan** [streamlink/streamlink#2009]
> 
> streamlink https://www.twitch.tv/vinesauce/video/296211543
> [cli][info] Found matching plugin twitch for URL https://www.twitch.tv/vinesauce/video/296211543
> error: Unable to open URL: https://clips.twitch.tv/api/v2/clips/video/status (404 Client Error: Not Found for url: https://clips.twitch.tv/api/v2/clips/video/status?as3=t)
> 

https://dev.twitch.tv/docs/api/reference/

Vods now no longer work with Streamlink as far as I can tell.

**Comment by skulblakka** [comment]
Duplicate of #1982 and fixed in #1988 

```
> streamlink https://www.twitch.tv/vinesauce/video/296211543
[cli][info] Found matching plugin twitch for URL https://www.twitch.tv/vinesauce/video/296211543
[plugin.twitch][info] Attempting to authenticate using OAuth token
[plugin.twitch][info] Successfully logged in as ########
Available streams: audio, 160p (worst), 360p, 480p, 720p, 720p60_alt, 720p60 (best)
```

**Comment by gravyboat** [comment]
Yep, closing as a duplicate, this was fixed already, we just haven't done a new release of the installer.
